docs/datascientist_onboard.md

```diff
@@ -0,0 +1,230 @@
+In this scenario, you should rebuild base image, then build job image. Firstly, find the base-image's Dockerfile `Dockerfile.build.base` at [base image directory](https://github.com/Microsoft/pai/tree/master/job-tutorial/Dockerfiles), e.g. change the cuda tag at [cuda9.0-cudnn7/Dockefile.build.base Line33](https://github.com/Microsoft/pai/blob/a59f05fe32934dd1164c26caafc0676f0763b692/job-tutorial/Dockerfiles/cuda9.0-cudnn7/Dockerfile.build.base#L33) , all the version tags can be found at [nvidia/cuda](https://github.com/Microsoft/pai/blob/a59f05fe32934dd1164c26caafc0676f0763b692/job-tutorial/Dockerfiles/cuda9.0-cudnn7/Dockerfile.build.base#L33).
+Then run the following command to build and push base image, replace the ${cuda_version} and ${cudnn_version} to the version you chose, replace ${your_user_name} to the user name of your docker registery:
+```
+docker build -f /path-to-dockerfile/Dockerfile.build.base -t ${your_user_name}/pai.build.base:hadoop2.7.2-cuda${cuda_version}-cudnn${cudnn_version}-devel-ubuntu16.04 Dockerfiles/
+
+docker login
+docker push ${your_user_name}/pai.build.base:hadoop2.7.2-cuda${cuda_version}-cudnn${cudnn_version}-devel-ubuntu16.04
+```
+To build job container from your customized base image, still use tensorflow for example, just replace the [Dockerfile.example.tensorflow Line19](https://github.com/Microsoft/pai/blob/a59f05fe32934dd1164c26caafc0676f0763b692/examples/tensorflow/Dockerfile.example.tensorflow#L19) to the tag of your customized base image's tag, then build and push the job container, the process is the same.
+	
+### 3. Write a job JSON configuration file 
+After you've prepared your data, code and job container, you need write a job configuration file in JSON format to describe the container image you use, the resources you requests,etc. Then submit the configuration file on PAI, it will allocate resources and lauch your job container according to the file.
+
+You can start from our [example directory](https://github.com/Microsoft/pai/tree/master/examples), there is a "*.json" configuraton file under each sub-directory for your reference.
+
+Or you can [write your customized configuration files](https://github.com/Microsoft/pai/blob/master/docs/job_tutorial.md#jobjson).
+	
+### 4. Register an account, then submit job
+Now you are ready to summit a job, find your admin to register a username and password, open PAI homepage, login to summit your job, refer to [Submit a job in Web Portal](https://github.com/Microsoft/pai/blob/master/docs/submit_from_webportal.md) for more details.
+
+Other submit ways:
+- [Submit a job in Visual Studio](https://github.com/Microsoft/vs-tools-for-ai/blob/master/docs/pai.md)
+
+- [Submit a job in Visual Studio Code](https://github.com/Microsoft/vscode-tools-for-ai/blob/master/docs/quickstart-05-pai.md)
+
+### 5. Debug job 
+Please refer to [debug job document](https://github.com/Microsoft/pai/blob/master/docs/job_tutorial.md#how-to-debug-the-job-) for details.
```

Comment:
Should we explain how to debug from logs? I don't think users know what the `YarnContainerScript.log` and `DockerContainerScript.log` mean. For me, they are very confusing until I read related scripts, but it is impossible for users.

I think we need to teach users how to judge the exit reason, if it's system reason, users should ask help from admin or DRI, otherwise where could users find their code stdout and stderr info.